Changelog.md

```diff
@@ -1,5 +1,8 @@
 ### 0.8.16 (unreleased)
 
+Important Bugfixes:
+ * Code Generation: Fix data corruption that affected ABI-encoding of calldata values represented by tuples: structs at any nesting level; argument lists of external functions, events and errors; return value lists of external functions. Up to 32 leading bytes of the first dynamically-sized value in the tuple would get zeroed when the last component contained a statically-sized array.
```

Comment:
```suggestion
 * Code Generation: Fix data corruption that affected ABI-encoding of calldata values represented by tuples: structs at any nesting level; argument lists of external functions, events and errors; return value lists of external functions. Up to 32 leading bytes of the first dynamically encoded value in the tuple would get zeroed when the last component contained a statically-sized array.
```
In changelog and bug list we did use that phrasing before and it's clearer that way that also fixed-size-arrrays of dynamically-sized arrays are affected.